dotCMS/src/main/java/com/dotmarketing/util/starter/ExportStarterUtil.java

```diff
@@ -354,8 +391,236 @@ private void createJSONFiles()  {
+            throw new DotRuntimeException(e);
+        }
+        return starterFiles;
+    }
+
+    /**
+     * Certain dotCMS database tables are not mapped to Java classes, but are required for dotCMS to start up
+     * correctly. This method handles the generation of JSON data for them.
+     *
+     * @return The list of {@link FileEntry} objects containing the records of the "special" dotCMS database tables.
+     */
+    @CloseDBIfOpened
+    private List<FileEntry> getAdditionalDataAsJSON()  {
+        final List<FileEntry> starterFiles = new ArrayList<>();
+        final ObjectMapper defaultObjectMapper = DotObjectMapperProvider.getInstance().getDefaultObjectMapper();
+        try {
+            List _list = ImmutableList.of(APILocator.getCompanyAPI().getDefaultCompany());
+            String contentAsJson = defaultObjectMapper.writeValueAsString(_list);
+            starterFiles.add(new FileEntry(Company.class.getName() + JSON_FILE_EXT, contentAsJson));
+
+            // Users
+            _list = APILocator.getUserAPI().findAllUsers();
+            _list.add(APILocator.getUserAPI().getDefaultUser());
+            contentAsJson = defaultObjectMapper.writeValueAsString(_list);
+            starterFiles.add(new FileEntry(User.class.getName() + JSON_FILE_EXT, contentAsJson));
+
+            final DotConnect dc = new DotConnect();
+
+            dc.setSQL("select * from counter");
+            _list = dc.loadResults();
+            contentAsJson = defaultObjectMapper.writeValueAsString(_list);
+            starterFiles.add(new FileEntry("Counter" + JSON_FILE_EXT, contentAsJson));
+
+            _list = ImageLocalManagerUtil.getImages();
+            contentAsJson = defaultObjectMapper.writeValueAsString(_list);
+            starterFiles.add(new FileEntry("Image" + JSON_FILE_EXT, contentAsJson));
+
+            dc.setSQL("select * from portlet");
+            _list = dc.loadResults();
+            contentAsJson = defaultObjectMapper.writeValueAsString(_list);
+            starterFiles.add(new FileEntry("Portlet" + JSON_FILE_EXT, contentAsJson));
+
+            final List<FileEntry> jsonFileEntries = new ContentTypeImportExportUtil().exportContentTypes();
+            starterFiles.addAll(jsonFileEntries);
+
+            contentAsJson = WorkflowImportExportUtil.getInstance().exportWorkflowsToJson();
+            starterFiles.add(new FileEntry("WorkflowSchemeImportExportObject" + JSON_FILE_EXT, contentAsJson));
+
+            contentAsJson = RulesImportExportUtil.getInstance().exportToJson();
+            starterFiles.add(new FileEntry("RuleImportExportObject" + JSON_FILE_EXT, contentAsJson));
+
+            Logger.info(this, "Additional exportable JSON files have been generated successfully!");
+        } catch (final Exception e) {
+            Logger.error(this, e.getMessage(), e);
+            throw new DotRuntimeException(e);
+        }
+        return starterFiles;
+    }
+
     private void moveAssetsToBackupDir() throws IOException{
```

Comment:
Can we get rid of this method?

Reply:
Right, I'm in the process of removing as much unnecessary code as possible. Done!